src/three-components/ModelScene.js

```diff
@@ -199,62 +182,25 @@ export default class ModelScene extends Scene {
     if (width !== this.width || height !== this.height) {
       this.width = Math.max(width, 1);
       this.height = Math.max(height, 1);
-      this.aspect = this.width / this.height;
       // In practice, invocations of setSize are throttled at the element level,
       // so no need to throttle here:
-      this.applyRoomSize();
+      this.applyCanvasSize();
     }
   }
 
-  /**
-   * Updates the 3D room and model scale based on the 2D
-   * dimensions for the encapsulating element.
-   */
-  applyRoomSize() {
+  applyCanvasSize() {
```

Comment:
If all this is doing is changing the dimensions of a `<canvas>`, then I would agree this name is close to fine (I might have gone with `updateCanvasSize`).

But, this is also computing values for the `framedHeight` and `modelDepth`. These seem only passingly related to changing the size of a `<canvas>`. All of these things are part of some higher-order change taking place, so we should probably choose a name that reflects that.

Reply:
Agreed. I kind of hate naming, so suggestions welcome.

Reply:
We use the word "frame" a lot these days. I would be happy with `updateFraming`.